### PR TITLE
fixed typos on pool-upgrade.md and updated readthedocs site index

### PR DESCRIPTION
**Pull request by michaeldboyd**
Signed-off-by: Michael Boyd <michaeldavisboyd@gmail.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

File: docs/source/index.rst [hunk 1]
```diff
@@ -11,18 +11,20 @@ Welcome to Hyperledger Indy Node's documentation!
    :maxdepth: 1
 
    transactions.md
-   requests.md
+   requests-new.md
```
Comment (ashcherbakov) [inline docs/source/index.rst]: The valid request description is in `requests.md`. `requests-new.md` is not implemented yet, so I think it doesn't make sense to include it into index.

File: docs/source/index.rst [hunk 2]
```diff
@@ -11,18 +11,20 @@ Welcome to Hyperledger Indy Node's documentation!
...
    add-node.md
    helper-scripts.md
    setup-iptables.md
    node-monitoring-tools-for-stewards.md
+   node-state-diagnostic-tools.md
    ci-cd.md
    indy-file-structure-guideline.md
    write-code-guideline.md
    setup-dev.md
+   1.3_to_1.4_migration_guide.md
```
Comment (ashcherbakov) [inline docs/source/index.rst]: The current Sovrin version is 1.9, so I don't think that we need to include this migration guide into index. It has no value for uses now.

**Comment by brentzundel** [comment]
@michaeldboyd if you address the requested changes, I can approve this PR and we may be able to move it forward.

**Comment by brentzundel** [comment]
No response for 3 months. Closing this PR.

**Pull request closed without merging**